backend/open_webui/utils/task.py

```
@@ -207,6 +207,24 @@ def title_generation_template(
     return template
 
 
+def follow_up_generation_template(
+    template: str, messages: list[dict], user: Optional[dict] = None
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)